docs/api.md

```diff
@@ -18,6 +18,16 @@ Provide your token via the `Private-Token` header:
 curl -H "Private-Token: TOKEN" https://us.prairielearn.com/pl/api/v1/<REST_OF_PATH>
 ```
 
+## Types of Endpoints
+
+When sending a request to an endpoint, it will require you to send either a `GET` or a `POST` request. A `GET` request retrieves information from PrairieLearn, such as gradebook information. Whereas a `POST` request asks PrairieLearn to perform an action. For example, syncing a course repository to PrairieLearn. You will need the correct permissions to be able to perform this request. Unless specified otherwise, all requests are going to be `GET` requests and will follow the above example format when using `curl`.
```

Comment:
Wordsmithing + including links to MDN:

```suggestion
API endpoints require either a [`GET` request](https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Methods/GET) or a [`POST` request](https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Methods/POST). A `GET` request retrieves information from PrairieLearn, such as gradebook information. A `POST` request asks PrairieLearn to perform an action, such as syncing a course GitHub repository. Unless otherwise specified, all requests are `GET` requests and will follow the above example format when using `curl`.
```

If we want to retain a mention of permissions, that should go elsewhere, that has nothing to do with discussions of HTTP methods.

Reply:
I think we can safely omit it.